common_sensor_launch/launch/velodyne_node_container.launch.py

```diff
@@ -138,13 +138,13 @@ def create_parameter_dict(*args):
 
     nodes.append(
         ComposableNode(
-            package="velodyne_pointcloud",
-            plugin="velodyne_pointcloud::Interpolate",
-            name="velodyne_interpolate_node",
+            package="pointcloud_preprocessor",
+            plugin="pointcloud_preprocessor::DistortionCorrectorComponent",
+            name="distortion_corrector_node",
             remappings=[
-                ("velodyne_points_ex", "mirror_cropped/pointcloud_ex"),
-                ("velodyne_points_interpolate", "rectified/pointcloud"),
-                ("velodyne_points_interpolate_ex", "rectified/pointcloud_ex"),
+                ("~/input/velocity_report", "/vehicle/status/velocity_status"),
```

Comment:
I think it's `~/input/twist` (geometry_msgs::msg::TwistWithCovarianceStamped), not `~/input/velocity_report`.

https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp#L47

Reply:
And, we should add `~/input/imu`.

https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp#L51

Reply:
Sorry I refeerd old version.
Fixed in https://github.com/autowarefoundation/sample_sensor_kit_launch/pull/41/commits/21a46b2f007ac839276c2c3c93a7b566ae8c3ad8